### PR TITLE
Run build on any docs change

### PR DESCRIPTION
**Pull request by jpignata**
🚨  **ACHTUNG**🚨 This may require some repo settings fiddling after merge.

*Issue #, if available:* n/a

*Description of changes:* To ensure the docs site has parseable content, run the build on every check-in after spellcheck.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jamesonwilliams — APPROVED** [review]

**Comment by jpignata** [comment]
@sammartinez @tjleing Let me know if you're interested in merging this. It will require some coordination.

**Comment by jpignata** [comment]
Ping.

**Review by tjleing — APPROVED** [review]
Sorry for seeing this so late -- thank you @jpignata for making this change and @ericclemmons for pointing it out to me!  This certainly seems like a necessary addition after those PRs with broken links.

**Comment by ericclemmons** [comment]
This is interesting. Spellcheck hasn't reported its status, so this PR is stuck!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: Test
 on: pull_request
 jobs:
-  spellcheck:
-    name: Spellcheck
+  build:
+    name: Build
```
Comment (ericclemmons) [inline .github/workflows/build.yml:5]: Can I trigger a rebuild with a noop suggestion?
```suggestion
    name: Build
```

Comment (ericclemmons) [reply .github/workflows/build.yml:5]: Nope!

**Pull request merged**